eBCSgen/Parsing/ParseBCSL.py

```diff
@@ -87,13 +87,14 @@ def to_side(self):
 
 
 GRAMMAR = r"""
-    model: rules (inits)? (definitions)? (complexes)? (regulation)?
+    model: (sections)* rules (sections | rules)*
```

Comment:
What is the reason for having `rules` there for the second time?

Reply:
To allow `rules` to be defined multiple times, just like `sections`. This keeps things consistent and fair."